drag_and_drop_v2/public/css/drag_and_drop_edit.css

```diff
@@ -315,8 +320,9 @@
 
 .xblock--drag-and-drop--editor .remove-item .icon.remove {
     background: #fff;
+    color: #0072a7;
 }
 .xblock--drag-and-drop--editor .remove-item .icon.remove:before,
 .xblock--drag-and-drop--editor .remove-item .icon.remove:after {
-    background: #2e83cd;
+    background: #236299;
```

Comment:
@bradenmacdonald Yes, I changed the colors to increase contrast. Using [this tool](http://webaim.org/resources/contrastchecker/) to check contrast values I found that many pairs of foreground and background colors did not pass the check for WCAG AA. To make the changes as subtle as possible I stopped lightening (or darkening) individual colors as soon as the tool reported that a given pair was passing for WCAG AA. For some pairs of colors adjusting the contrast involved a larger number of steps than for others. That's why some of the colors look similar but have slightly different values.